File: examples/php/fibonacci.php
<?php
function fib($n) {
    $a = 0;
    $b = 1;
    $i = 0;
    while ($i < $n) {
        $c = $a + $b;
        $a = $b;
        $b = $c;
        $i = $i + 1;
    }
    return $a;
}

$i = 0;
while ($i < 10) {
    print(fib($i) . "\n");
    $i = $i + 1;
}

File: examples/rust/fibonacci.rs
fn fib(n: i64) -> i64 {
    let mut a = 0;
    let mut b = 1;
    for i in 0..n {
        let next = a + b;
        a = b;
        b = next;
    }
    return a;
}

fn main() {
    for i in 0..10 {
        println!("{}", fib(i));
    }
}
